gateway/transport/server.go

```diff
@@ -106,9 +106,26 @@ func (s *Server) PreConnect(ctx context.Context, req *pb.PreConnectRequest) (*pb
 	return resp, nil
 }
 
+func GetAccessModesFromConnect(clientVerb string, clientOrigin string) string {
+	switch {
+	case clientVerb == pb.ClientVerbExec && clientOrigin == pb.ConnectionOriginClient:
+		return "exec"
+	case clientVerb == pb.ClientVerbExec && clientOrigin == pb.ConnectionOriginClientAPI:
+		return "exec"
```

Comment:
Since you're returning the same value for distinct statements in a case block, just use comma to separate expressions and return once.

Example:

```go
switch {
	case clientVerb == pb.ClientVerbExec && clientOrigin == pb.ConnectionOriginClient,
               clientVerb == pb.ClientVerbExec && clientOrigin == pb.ConnectionOriginClientAPI:
		return "exec"
         (...)
```